config/universal-redux.config.js

```diff
@@ -45,6 +45,14 @@ module.exports = (projectRoot, sourceRoot) => {
       // Expects: Number
       */
       // maxAge: 0
+
+      /*
+      // The renderer returns middleware for universal rendering. This config option sets the framework to
+      // which this middleware should be compatible.
+      // Expects: String
+      // options: 'express' || 'koa'
+       */
+      rendererWebFramework: 'express'
```

Comment:
@bartolkaruza I'd prefer to just call this `webFramework` if that's OK with you?